presidio-analyzer/presidio_analyzer/nlp_engine/spacy_nlp_engine.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, models: Optional[Dict[str, str]] = None):
         logger.debug(f"Loading SpaCy models: {models.values()}")
 
         self.nlp = {
-            lang_code: spacy.load(model_name, disable=["parser", "tagger"])
+            lang_code: spacy.load(model_name, disable=["parser"])
```

Comment:
Intentional?

Reply:
Yes this is a bug we have in our existing release. lemmas (context words) don't work. Actually deserves a different PR.